.github/workflows/run-unit-tests.yml

```diff
@@ -3,8 +3,20 @@ name: Run unit tests
 on:
   push:
     branches: [develop, master]
+    paths-ignore:
+      - './github/**'
+      - 'docs/**'
+      - 'test/e2e/**'
```

Comment:
The path mentioned here is the end to end test and we dont want to skip the execution. Could you please check again.

Reply:
sure thanks, I understand from [this task](https://github.com/MetaMask/MetaMask-planning/issues/2676) is there was a desire to not running unit-tests where only e2e files are changing but I was also not sure skipping unit-tests. I can remove this.

Reply:
I think the main request here is to skip the execution of the E2E tests when there are changes only in the unit tests, and vice versa. You first need to determine the changed files and then depending on the changes the corresponding action needs to be taken. 

Note - The condition is that the action should be triggered only if there is a change in the specific type of test only files.

Reply:
This `paths-ignore` thing supposed to skip unit-tests workflow if the change is only in `tests/e2e/**` if there is any other change somewhere else it should still run unit-tests based on my understanding from the docs but I'll do some tests in a dummy repository to make sure its working as expected. 

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore